lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -260,6 +262,23 @@ void LocalEnforcer::aggregate_records(
   }
   complete_termination_for_released_sessions(
       session_map, sessions_with_reporting_flows, session_update);
+  cleanup_dead_sessions(dead_sessions_to_cleanup);
+}
+
+void LocalEnforcer::cleanup_dead_sessions(
+    const RuleRecordSet dead_sessions_to_cleanup) {
+  if (dead_sessions_to_cleanup.size() == 0) {
+    return;
+  }
+  MLOG(MINFO) << "Deactivating" << dead_sessions_to_cleanup.size()
+              << " flows that are attached to non-existent sessions";
+  Teids empty_teids;
```

Comment:
What about what you mentioned of adding the teid field on the rule request?
This way we can start closing the loop for those actions.

Reply:
oh yeah good call :) one less thing to worry about.